recipe/meta.yaml

```diff
@@ -17,23 +18,23 @@ build:
     - vc10  # [win and py34]
     - vc14  # [win and py>34]
   skip: True  # [py<34 or win]
```

Comment:
The windows build issues with 0.0.8 seem to have been resolved upstream, so we should try to enable it for this release.